ddsketch/mapping.py

```diff
@@ -99,7 +97,15 @@ def _log_gamma(self, value):
         return math.log2(value) * self._multiplier
 
     def _pow_gamma(self, value):
-        return np.exp2(value / self._multiplier)
+        return math.pow(2, value / self._multiplier)
```

Comment:
~~~
In [4]: assert 2 ** 10.3 == math.pow(2, 10.3)

In [5]: %timeit 2 ** 10.3
7.25 ns ± 0.134 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

In [6]: %timeit math.pow(2, 10.3)
82.6 ns ± 0.386 ns per loop (mean ± std. dev. of 7 runs, 10000000 loops each)
~~~

Reply:
Nice, great catch!